docs/Tutorial.md

```diff
@@ -40,7 +41,7 @@ Backend extension should be installed _after_ installing ONE-vscode extension.
 ### Use ONE-vscode without any backend extension
 ONE-vscode supports `ONE` backend by default. This requires [Docker Installation](https://docs.docker.com/desktop/install/ubuntu/) and [root privileges](https://docs.docker.com/engine/install/linux-postinstall/). However it supports only basic compilation. If you want to compile for a specific hardware model, please install backend extensions.
 
-<img src="./OneToolchain.jpg" width=1000/>
+![](OneToolchain.jpg)
```

Comment:
Thanks for your contribution! BTW, `img` tag was used to set width of the figure. If it's replaced with `![]()`, the tutorial page will look different.